### PR TITLE
Add/Remove installer options to fix the installer test

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
```
Launching pytest with arguments -k test_installer_options_and_sections /Users/okhatavk/satellite/robottelo/tests/foreman/installer/test_installer.py --no-header --no-summary -q in /Users/okhatavk/satellite/robottelo

============================= test session starts ==============================
collecting ... collected 18 items / 16 deselected / 2 selected

tests/foreman/installer/test_installer.py::test_installer_options_and_sections[params] 
tests/foreman/installer/test_installer.py::test_installer_options_and_sections[sections] 

================ 2 passed, 16 deselected, 2 warnings in 30.15s =================
```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
waiting for the developer's approval

**Review by devendra104 — APPROVED** [review]
ACK

**Review by Gauravtalreja1 — APPROVED** [review]

**Pull request merged**
